docs/freqtrade-101.md

```diff
@@ -0,0 +1,188 @@
+
+- **long**: You buy the coin based on a stake, e.g. buying the coin BTC using USDT as your stake, and you make a profit by selling the coin at a higher rate than you paid for. Profits are made in long trades by the coin value going up versus the stake.
+- **short**: You borrow capital from the exchange in the form of the coin, and you pay back the stake value of the coin later. Profits are made in short trades by the coin value going down versus the stake (you pay the loan off at a lower rate).
+
+Whilst Freqtrade supports spot and futures markets for certain exchanges, for simplicity we will focus on spot (long) trades only.
+
+## Structure of a Basic Strategy
+
+### Main dataframe
+
+Freqtrade strategies use a tabular data structure with rows and columns known as a `dataframe` to generate signals to enter and exit trades.
+
+Each pair in your configured pairlist has its own dataframe. Dataframes are indexed by the `date` column, e.g. `2024-06-31 12:00`.
+
+The next 5 columns represent the `open`, `high`, `low`, `close` and `volume` (OHLCV) data.
+
+### Populate indicator values
+
+The `populate_indicators` function adds columns to the dataframe that represent the technical analysis indicator values.
+
+Examples of common indicators include Relative Strength Index, Bollinger Bands, Money Flow Index, Moving Average, and Average True Range.
+
+Columns are added to the dataframe by calling technical analysis functions, e.g. ta-lib's RSI function `ta.RSI()`, and assigning them to a column name, e.g. `rsi`
+
+```python
+dataframe['rsi'] = ta.RSI(dataframe)
+```
+
+??? Hint "Technical Analysis libraries"
+    Different libraries work in different ways to generate indicator values. Please check the documentation of each library to understand
+    how to integrate it into your strategy. You can also check the [Freqtrade example strategies](https://github.com/freqtrade/freqtrade-strategies) to give you ideas.
+
+### Populate entry signals
+
+The `populate_entry_trend` function defines conditions for an entry signal.
+
+The dataframe column `enter_long` is added to the dataframe, and when a value of `1` is in this column, Freqtrade sees an entry signal.
+
+??? Hint "Shorting"
+    To enter short trades, use the `enter_short` column.
+
+### Populate exit signals
+
+The `populate_exit_trend` function defines conditions for an exit signal.
+
+The dataframe column `exit_long` is added to the dataframe, and when a value of `1` is in this column, Freqtrade sees an exit signal.
+
+??? Hint "Shorting"
+    To exit short trades, use the `exit_short` column.
+
+## A simple strategy
+
+Here is a minimal example of a Freqtrade strategy:
+
+```python
+from freqtrade.strategy.interface import IStrategy
+from pandas import DataFrame
+import talib.abstract as ta
+
+class MyStrategy(IStrategy):
+
+    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
```

Comment:
I think at least `stoploss`, and maybe `minimal_roi` should be added. Without stoploss, the strategy won't start unless you're using a non-default config (with stoploss configured there).
This should however be a running strategy in my eyes...